### ISSUE TITLE
Loss landscape visualization

### ISSUE THREAD

**Issue by chenrxi** [debcaldarola/fedsam#2]
![image](https://user-images.githubusercontent.com/60959000/192463323-d7dc9552-4b79-49c2-a8f8-90bb71e29fb0.png)
how do you put three different models' train loss surface together. By the way,  does train loss refer to the cross-entropy loss?

**Comment by debcaldarola** [comment]
Hi, 
thanks for your interest! 
You can find all the implementation details in Appendix C.3 of our paper, together with references to previous works. As for the code, we adapted Garipov's repo to our scenario (https://github.com/timgaripov/dnn-mode-connectivity). 
Yes, the loss we refer to is always the cross-entropy loss.
Hope this helps you!

**Comment by chenrxi** [comment]
Hi, how do you implementation computation of sharpness of a minima?
